### DISCUSSION TITLE
Setup instructions with local music and later re-flashing

### DISCUSSION THREAD

**Discussion by mrManniManfred** [rern/rAudio#474] [General]
Hi everybody,

what would be a good solution to setup a microSD-card with rAudio-1 when storing local music and wanting to update /flash rAudio from time to time without interfering with the stored music?

So only flashing the rAudio-part to a new version and having some kind of seperate partition with my music.

Any ideas?

regards
Manni :)

**Comment by ceejayemm** [comment]
Manni

I would flash rAudio to the SD card (and nothing else) and then hold your music on a separate USB or NAS drive. In that way you can re-flash the SD card whenever you want , and if you keep a good backup before doing so, you can easily restore your settings either during the re-flash process or restore the backup separately after the new card is ready to use.

Chris
